mjml/mjml2html.py

```diff
@@ -113,6 +114,11 @@ def default_attr_classes(value):
             )
 
             _parse_mjml = lambda mjml: parse(mjml, nextParentMjClass)
+            # XXX: ugly - but need raw XML content for mj-raw tag
+            if tagName == 'mj-raw':
+                from lxml.etree import tostring
+                content = tostring(_mjml).strip().decode('utf-8')
+                content = content.replace('<mj-raw>', '').replace('</mj-raw>', '')
```

Comment:
Not sure what `lxml.etree.tostring()` will return but this seems very specific. Does it handle something like `<mj-raw >`? (extra space)

Also this is basically about stripping the `<mj-raw>...</mj-raw>` container, right? Is there no functionality directly in lxml which we could use here?

Reply:
> Not sure what `lxml.etree.tostring()` will return but this seems very specific. Does it handle something like `<mj-raw >`? (extra space)

the ```.strip()```after could be removed - on the other side extra space shouldn't matter in the final HTML document

> Also this is basically about stripping the `<mj-raw>...</mj-raw>` container, right? Is there no functionality directly in lxml which we could use here?

correct, afaik there is no such function within lxml  - but there are multiple approaches to solve it:

https://stackoverflow.com/questions/4624062/get-all-text-inside-a-tag-in-lxml
https://stackoverflow.com/questions/11122397/how-do-i-get-the-whole-content-between-two-xml-tags-in-python

I don't mind which one to use if we have to - up to you to decide - and thats the reason the whole paragraph has a ```XXX: ugly``` above ;)

Reply:
Thank you for the pointers. Not sure which version is easier to grasp. Also the comments mention a few edge-case bugs so we should definitively create a small utility function which we can test independently of the full mjml machinery.

I'll merge this PR as-is (it enhances the status quote and the part above is not too bad imho) and create ticket for this specific issue.